functions/src/converter.ts

```diff
@@ -0,0 +1,79 @@
+import * as yargs from 'yargs';
+
+//converter part 1
+
+const args = yargs.option('inputPath', {alias: 'i', demand: true, type: 'string'}).option('outputPath',
+               {alias: 'o', demand: true, type:'string'}).argv;
+const fs = require('fs');
+const rawJSON = fs.readFileSync(args.inputPath);
+const dataJSON = JSON.parse(rawJSON);
+convertToDBSchema(dataJSON);
```

Comment:
what's the purpose of having this function? you could just inline everything its doing to this script. The function itself is doing more than just the conversion, its performing the actual io of writing the converted file to disk and sending a message to the console. 

Typically you want to try to write functions that are "pure", i.e. perform no side effects like network calls or io, and always behave the same depending on the function's input, and lift all "impure" work to the highest level possible. If you have the time, I'd recommend watching [this talk](https://www.destroyallsoftware.com/screencasts/catalog/functional-core-imperative-shell) on the benefits of such an approach